BNN.py

```diff
@@ -52,49 +48,54 @@ def new_bnn(log_divergence_weight=-10, prior_stddev=1, residual_weight=0.1):
             bias_divergence_fn=weighted_divergence_fn(log_divergence_weight))
     ])
 
-#training-data:
-x = np.array([[1., 2., 3.], [2., 3., 4.]], dtype=np.float32)
-y = np.array([1, 5], dtype=np.float32)
-# print("Shape of x", x.shape)
-y.shape[0]
+if __name__ == "__main__":
```

Comment:
I moved all of the import side-effects to a `main` codeblock so that these are only run when this file is explicitly called as the main file.